packages/bpk-component-bottom-sheet/src/BpkBottomSheet.module.scss

```diff
@@ -17,16 +17,156 @@
+  overflow: hidden;
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+
+  // FIX: Prevents ugly flickering when tapping inside the modal in Mobile Safari
+  -webkit-tap-highlight-color: transparent;
+
+  @include shadows.bpk-box-shadow-xl;
+  @include radii.bpk-border-radius-sm;
+
+  @include breakpoints.bpk-breakpoint-mobile {
+    position: fixed;
+    bottom: 0;
+    height: fit-content;
+    max-height: 90%;
+    margin-bottom: 0;
+    border-radius: tokens.$bpk-border-radius-lg tokens.$bpk-border-radius-lg 0 0;
+    overflow-x: hidden;
+  }
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  &--appear {
+    animation-duration: 0.24s;
+    animation-name: slide-up;
+    animation-timing-function: ease-in-out;
+
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      transform: scale(0.9);
+      opacity: tokens.$bpk-modal-initial-opacity;
+      animation: none;
+    }
+  }
+
+  &--appear-active {
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      transform: scale(1);
+      opacity: tokens.$bpk-modal-opacity;
+    }
+  }
+
+  &--exit {
+    animation-direction: reverse;
+    animation-duration: 0.24s;
+    animation-name: slide-up;
+    animation-timing-function: ease-in-out;
+
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      animation: none;
+    }
+  }
+
+  &--content {
+    padding: tokens.$bpk-modal-content-padding;
+    flex: 1;
+    overflow-y: auto;
+  }
+
+  &--wide {
+    @include breakpoints.bpk-breakpoint-above-mobile {
+      max-width: tokens.$bpk-modal-wide-max-width;
+    }
+  }
+
+  &--header {
+    position: sticky;
+    top: 0;
+    z-index: tokens.$bpk-zindex-tooltip - 1;
+
+    @include borders.bpk-border-bottom-sm(tokens.$bpk-line-day);
+  }
+
+  &--navigation {
     display: flex;
-    padding: tokens.bpk-spacing-lg();
+    justify-content: space-between;
+    background-color: tokens.$bpk-modal-background-color;
+  }
+
+  &--heading {
+    width: calc(
+      100% - ((#{$close-button-width}) * 2 + #{tokens.bpk-spacing-sm()} / 2)
+    ); // Treat title as if it was always surrounded by close and action to center align it
+
+    margin-right: auto;
+    text-align: center;
+
+    @include typography.bpk-text;
+    @include typography.bpk-label-1;
+  }
+
+  &--close-button {
+    position: relative; // Override position: absolute coming from BpkNavigationBar
+    left: auto;
+    margin-right: tokens.bpk-spacing-lg();
+
+    @include utils.bpk-rtl {
+      right: auto;
+      margin-right: calc(-#{tokens.bpk-spacing-sm()} / 2);
+      margin-left: tokens.bpk-spacing-lg();
+    }
+  }
+
+  &--action-button {
+    position: relative; // Override position: absolute coming from BpkNavigationBar
+    right: auto;
+    width: calc(#{$close-button-width} - (#{tokens.bpk-spacing-sm()}));
+    margin-right: calc(#{tokens.bpk-spacing-sm()} / 2);
+    margin-left: calc(#{tokens.bpk-spacing-sm()});
+    word-break: break-all;
```

Comment:
We might want to check if we want to expect this behaviour!

Reply:
Yes this is confirmed with design 👍